### ISSUE TITLE
Upgrade to Ubuntu Xenial

### ISSUE THREAD

**Issue by flibbertigibbet** [azavea/cac-tripplanner#1100]
~~HVM [AMIs for Ubuntu](https://cloud-images.ubuntu.com/locator/ec2/) Trusty are no longer available, blocking redeploy.~~

Attempting to use a paravitualied AMI instead results in the error:
```
Virtualization type 'hvm' is required for instances of type 'm4.large'
```

Upgrade the AMI and vagrant box OS versions to Xenial. Fix ansible roles and service scripts for the upgrade, as done [here](https://github.com/azavea/cicero/commit/ce6d0964ec44736cb4dd77f746744ec140790c92).

**Comment by hectcastro** [comment]
Is this the Ubuntu AMI lookup process used in the Packer AMI build process?

We have an alternate Trusty AMI lookup procedure that uses the AWS API. That was working in other projects as of Friday.

- https://github.com/azavea/cicero/blob/develop/deployment/packer/driver.py#L18-L28
- https://github.com/WikiWatershed/model-my-watershed/blob/develop/deployment/packer/driver.py#L16-L27

I'd like to try porting that lookup process over before going through a Xenial upgrade. Also wouldn't mind the Xenial upgrade if if fits within the available budget.

**Comment by flibbertigibbet** [comment]
Yes. Cicero has been upgraded to Xenial already, but the second link to Model My Watershed looks like it should work to find an HVM Trusty AMI, off under a Canonical account.

Thinking it would be necessary, I started looking at what would be involved in upgrading just the OpenTripPlanner server to Xenial, but it looks like the service startup scripts need to be converted. I tried doing so in azavea/ansible-opentripplanner#35, but it didn't seem to be working properly with AuthBind in this project (the service starts, but isn't serving on the expected VM address.) There is no budget available for additional support work now.

Hopefully the Model My Watershed AMI lookup URL will work.
